domains/Discussions/Controllers/QuestionsUpdateController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Domains\Discussions\Controllers;
+
+use App\Http\Controllers\Controller;
+use Domains\Discussions\Models\Question;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class QuestionsUpdateController extends Controller
+{
+    private Question $questions;
+
+    public function __construct(Question $questions)
+    {
+        $this->questions = $questions;
+    }
+
+    public function __invoke(int $questionId, Request $request): Response
+    {
+        $question = $this->questions->newModelQuery()->findOrFail($questionId);
+
+        if ($request->user()->cannot('update', $question)) {
+            abort(Response::HTTP_FORBIDDEN);
+        }
+
+        $this->validate($request, [
+            'title' => ['required', 'string', 'max:255'],
+            'description' => ['nullable', 'string'],
+        ]);
+
+        $question->update([
+            'title' => $request->input('title'),
+            'description' => $request->input('description') ?? $question->description,
```

Comment:
The second argument of the `input()` method accepts a default value for when that input is not present in the request, so we can replace the use of the null coalescing operator with that.